dapps/marketplace/test/setup.js

```diff
@@ -3,20 +3,58 @@
  * the --file option to mocha. It sets up and tears down the infrastructure
  * (ethereum test node and IPFS) required to run tests.
  */
-
+import Web3 from 'web3'
+import bip39 from 'bip39'
+import hdkey from 'ethereumjs-wallet/hdkey'
 import services from './_services'
 
 const isWatchMode = process.argv.some(arg => arg === '-w' || arg === '--watch')
 
 let browser, webpackProcess, shutdownServices
 
+async function mnemonicToAddress(mnemonic) {
+  const seed = await bip39.mnemonicToSeed(mnemonic)
+  const masterKey = hdkey.fromMasterSeed(seed)
+  const masterWallet = masterKey.getWallet()
+  return masterWallet.getChecksumAddressString()
+}
+
 before(async function() {
   this.timeout(60000)
 
   shutdownServices = await services()
 
   browser = shutdownServices.extrasResult.browser
   webpackProcess = shutdownServices.extrasResult.webpackProcess
+
+  /**
+   * Need to fund the master account for the Relayer Purse
+   */
+  const web3 = new Web3('http://localhost:8545')
+  const netId = await web3.eth.net.getId()
+
+  if (parseInt(netId) === 999) {
+    if (!process.env.FORWARDER_MNEMONIC) {
```

Comment:
Can we default this if `process.env.NODE_ENV === 'test'`? Then we can skip on the extra env var in the package.json scripts